njoy_processor/Utils_ReadNJOYOutput.py

```diff
@@ -360,38 +368,68 @@ def ReadNJOYfile(njoy_filename = "output", verbose = False):
                 cross_sections["delayed_chi"] = \
                     ProcessDelayedChi(nL, file_lines)
 
+###---------matrix ------------------------------------------------------------
+            # caveat: the pp values from transfer(mf26) = 2x the pp from the xsec(mf23)
+            # caveat: the n,2n values from transfer(mf8/mt16) = 2x the n,2n from the xsec(mf3/mt16)
             if words[num_words - 1] == "matrix":
-                transfer_matrices[words[num_words - 3]] = \
+                particule_type = words[num_words - 2]
```

Comment:
`particule_type` is this `particle_type` with a french accent?

Reply:
yes. I can change that.